lua/soulbound_abilities/beebarrel.lua

```diff
@@ -0,0 +1,137 @@
+local ABILITY = {}
+
+ABILITY.Name = "Place Bee Barrel"
+ABILITY.Id = "beebarrel"
+ABILITY.Description = "Place a bee barrel that will release bees when it explodes"
+ABILITY.Icon = "vgui/ttt/roles/sbd/abilities/icon_beebarrel.png"
+
+local beebarrel_uses = CreateConVar("ttt_soulbound_beebarrel_uses", "3", FCVAR_REPLICATED, "How many uses of the place bee barrel ability should the Soulbound get. (Set to 0 for unlimited uses)", 0, 10)
+local beebarrel_bees = CreateConVar("ttt_soulbound_beebarrel_bees", "3", FCVAR_REPLICATED, "How many bees per beebarrel", 1, 10)
+local beebarrel_cooldown = CreateConVar("ttt_soulbound_beebarrel_cooldown", "0", FCVAR_NONE, "How long should the Soulbound have to wait between uses of the place bee barrel ability", 0, 10)
+
+table.insert(ROLE_CONVARS[ROLE_SOULBOUND], {
+    cvar = "ttt_soulbound_beebarrel_uses",
+    type = ROLE_CONVAR_TYPE_NUM,
+    decimal = 0
+})
+table.insert(ROLE_CONVARS[ROLE_SOULBOUND], {
+    cvar = "ttt_soulbound_beebarrel_cooldown",
+    type = ROLE_CONVAR_TYPE_NUM,
+    decimal = 1
+})
+
+if SERVER then
+
+    local function BeebarrelDamage(target, dmginfo)
+        if target:GetClass() == "prop_physics" then
+            
+            local model = target:GetModel()
+<<<<<<< HEAD
```

Comment:
These are failed merge artifacts that need to be cleaned up

Reply:
sorry, forgot about fetching, I think this issue is fixed now?